asmcli/commands/create-mesh.sh

```diff
@@ -22,8 +22,13 @@ parse_cluster_args() {
   fi
 
   while [[ $# != 0 ]]; do
-    local CLUSTER; CLUSTER="${1}"
-    context_append "clustersInfo" "${CLUSTER//\// }"
+    if [[ "$1" =~ .+/.+/.+ ]]; then
```

Comment:
I think we should check the opposite thing: if `$1` is a file, then we treat it like a kubeconfigFile, otherwise assume it's a clustersInfo. I don't want to mistake `asmcli create-mesh path/to/kubeconfig` as the wrong thing.

Reply:
ah good point, thanks!